apps/pdc-frontend/src/components/SearchBar/index.tsx

```diff
@@ -62,11 +64,15 @@ export const SearchBar: React.FC<SearchBarProps> = ({
     }
   };
 
+  const searchSegment = t('segments.search', {
+    defaultValue: 'zoeken',
```

Comment:
Zou je de URL segment strings willen verzamelen in één file, of één file per locale, en importeren? `import { SearchSegment } from 'UrlSegmentNL`;`

Als je voor de URL utility functions kiest, dan moet het misschien net iets anders werken.